test/script/e2e_run_byo.sh

```
@@ -83,7 +83,7 @@ kubectl create secret generic "$transport_secret" -n "$target_namespace" --kubec
 echo "transport secret is ready in $target_namespace namespace!"
 
 ## run e2e
-bash "$CURRENT_DIR/e2e_run.sh" -n $target_namespace -f "e2e-test-localpolicy,e2e-test-grafana,e2e-test-local-agent"
+bash "$CURRENT_DIR/e2e_run.sh" -n $target_namespace -f "e2e-test-localpolicy,e2e-test-grafana,e2e-test-transport-byo,e2e-test-local-agent"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)